src/assets/toolkit/styles/components/cms-content.css

```diff
@@ -9,3 +9,138 @@
   margin-right: calc(-1 * var(--cms-content-space));
   margin-left: calc(-1 * var(--cms-content-space));
 }
+
+/**
+ * Jetpack Device Lab Form Styles
+ */
+
+:root {
+  --JetpackInput-background-color: var(--color-white);
+  --JetpackInput-border-color: var(--color-gray);
+  --JetpackInput-border-color-focus: var(--color-blue);
+  --JetpackInput-border-radius: var(--control-radius);
+  --JetpackInput-border-width: var(--control-stroke);
+  --JetpackInput-color: var(--base-color);
+  --JetpackInput-disabled-opacity: var(--control-disabled-opacity);
+  --JetpackInput-padding: 0.5em 0.75em;
+  --JetpackInput-transition-duration: var(--motion-duration-lg);
+  --JetpackInput-transition-timing-function: var(--motion-timing-function-default);
+}
+
+/**
+ * 1. Most occurrences of this benefit from occupying the full width.
+ * 2. We aren't using a slightly more polished technique like text-indent due to
+ *    cross-browser edge cases where the rule would be ignored or would conflict
+ *    with platform-specific input type controls (search decorations, etc.).
+ * 3. Same `line-height` as `.Button` for easy alignment.
+ * 4. Unfortunately, Webkit won't play nice with our styles on some input types
+ *    unless we override its appearance.
+ * 5. Overrides specific `max-width` set in the JetPack style sheet.
+ */
+.CmsContent .comment-form input {
+  display: block; /* 1 */
+  box-sizing: border-box;
+  width: 100%; /* 1 */
+  max-width: 100%; /* 5 */
+  padding: var(--JetpackInput-padding);
+  font: inherit; /* 2 */
+  line-height: normal; /* 3 */
+  color: var(--JetpackInput-color);
+  border: var(--JetpackInput-border-width) solid var(--Input-border-color);
+  border-radius: var(--JetpackInput-border-radius);
+  background: var(--JetpackInput-background-color);
+  transition: all var(--JetpackInput-transition-duration) var(--Input-transition-timing-function);
+  -webkit-appearance: none; /* 4 */
+}
+
+.CmsContent .comment-form input:matches(:active, :hover) {
+  border-color: color(var(--JetpackInput-border-color) l(-15%));
+}
+
+.CmsContent .comment-form input:focus {
+  border-color: var(--JetpackInput-border-color-focus);
+  outline: none;
+}
+
+/**
+ * Jetpack Form Labels
+ */
+
+:root {
+  --JetpackLabel-color: var(--color-blue);
```

Comment:
Why are labels blue? I think they're blue for float labels, but default normally.